apps/emqx_rule_engine/src/emqx_rule_engine.erl

```diff
@@ -675,16 +675,17 @@ init_resource(Module, OnCreate, ResId, Config) ->
     emqx_rule_registry:add_resource_params(ResParams).
 
 init_resource_with_retrier(Module, OnCreate, ResId, Config) ->
-    try
-        Params = Module:OnCreate(ResId, Config),
-        ResParams = #resource_params{id = ResId,
-                                    params = Params,
-                                    status = #{is_alive => true}},
-        emqx_rule_registry:add_resource_params(ResParams)
-    catch Class:Reason:ST ->
-        emqx_rule_monitor:ensure_resource_retrier(ResId),
-        erlang:raise(Class, {init_resource, Reason}, ST)
-    end.
+    Params =
+        try
+            Module:OnCreate(ResId, Config)
+        catch Class:Reason:ST ->
```

Comment:
Do not log stacktrace when it’s a throw?

Reply:
fixed